test/suites/integration/tags_test.go

```diff
@@ -55,14 +56,14 @@ var _ = Describe("Tags", func() {
 				return ni.NetworkInterfaceId
 			})...)
 
-			Expect(tagMap(instance.Tags)).To(HaveKeyWithValue("TestTag", "TestVal"))
+			Expect(instance.Tags).To(ContainElement(&ec2.Tag{Key: lo.ToPtr("TestTag"), Value: lo.ToPtr("TestVal")}))
 			for _, volume := range volumes {
-				Expect(tagMap(volume.Tags)).To(HaveKeyWithValue("TestTag", "TestVal"))
+				Expect(volume.Tags).To(ContainElement(&ec2.Tag{Key: lo.ToPtr("TestTag"), Value: lo.ToPtr("TestVal")}))
```

Comment:
Does asserting that we have the element's pointer get us into any issues here, or does containElement go into the address for us

Reply:
Nope, it checks the address so we don't run into any issues here. The tests pass :)